services/app-api/form/defaultFormConfig.js

```diff
@@ -13,3 +13,5 @@ export const defaultProposedEffectiveDateSchema = [
 ];
 
 export const defaultTitleSchema = Joi.string().required();
+
+export const defaultWaiverAuthoritySchema = Joi.string().required();
```

Comment:
I like how you moved this to a common schema definition to enforce consistency for future changes!